lib/pymedphys/tests/e2e/yarn.lock

```diff
@@ -969,11 +969,16 @@ ms@2.0.0:
   resolved "https://registry.yarnpkg.com/ms/-/ms-2.0.0.tgz#5608aeadfc00be6c2901df5f9861788de0d597c8"
   integrity sha1-VgiurfwAvmwpAd9fmGF4jeDVl8g=
 
-ms@2.1.2, ms@^2.1.1:
+ms@2.1.2:
   version "2.1.2"
   resolved "https://registry.yarnpkg.com/ms/-/ms-2.1.2.tgz#d09d1f357b443f493382a8eb3ccd183872ae6009"
   integrity sha512-sGkPx+VjMtmA6MX27oA4FBFELFCZZ4S4XqeGOXCv68tT+jb3vk/RyaKWP0PTKyWtmLSM0b+adUTEvbs1PEaH2w==
 
+ms@^2.1.1:
```

Comment:
I'm not familiar with yarn or the yarn lock file, but it looks like some kind of mapping has changed, where a range of versions were going to 2.1.2 before, but now if a specific request for 2.1.2 is made, that's what is provided, but if a request for 2.1.1 or later is requested, 2.1.3 is used.
Unless there is something weird about 2.1.2, I think it would be easier to just have 2.1.3 and "that's what you get".

Reply:
This lock file isn't manually produced. Essentially it's just the same as a `poetry.lock` file, except instead of PyPI being the package source, it's NPM.

So in this case, the goal is to accept what yarn determines when an update is run.